<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSimuForward</class>
 <widget class="QDialog" name="DialogSimuForward">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>模拟正演参数设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="20,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox 
{border: 2px solid #0000EE;
 border-radius: 5px;
margin-top: 1ex;
}

QGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
}</string>
       </property>
       <property name="title">
        <string>参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>电位装置选择开关：         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_selectionSwitch">
            <item>
             <property name="text">
              <string>Schlumberger</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AMNB</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>供电电极附近的电阻率值：   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_sgM0Value">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_CII">
            <property name="text">
             <string>电流大小CII：              </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_CII">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>电流大小II：               </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_II">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>供电电源位置XPos：         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_currentINumX">
            <property name="text">
             <string>11</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>供电电源位置YPos：         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_currentINumY">
            <property name="text">
             <string>11</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>供电电源位置ZPos：         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_currentINumZ">
            <property name="text">
             <string>0</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>异常体单元节点XMin：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverX1">
              <property name="text">
               <string>9</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_begin_x">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>异常体单元节点XMax：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverX2">
              <property name="text">
               <string>14</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_end_x">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>异常体单元节点YMin：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverY1">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_begin_y">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>异常体单元节点YMax：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverY2">
              <property name="text">
               <string>15</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_end_y">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>异常体单元节点ZMin：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverZ1">
              <property name="text">
               <string>22</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_begin_z">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,1">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>异常体单元节点ZMax：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_inverZ2">
              <property name="text">
               <string>27</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>异常体实际位置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_actual_end_z">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverX1</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationXMinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverY1</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationYMinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverZ1</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationZMinChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverX2</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationXMaxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverY2</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationYMaxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_inverZ2</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogSimuForward</receiver>
   <slot>onActualLocationZMaxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActualLocationXMaxChanged()</slot>
  <slot>onActualLocationYMaxChanged()</slot>
  <slot>onActualLocationZMaxChanged()</slot>
  <slot>onActualLocationXMinChanged()</slot>
  <slot>onActualLocationYMinChanged()</slot>
  <slot>onActualLocationZMinChanged()</slot>
 </slots>
</ui>
